test/datadog/lambda/metrics.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockLength
+require 'datadog/lambda'
+require 'datadog/statsd'
+require 'datadog/lambda/metrics'
+require 'datadog/lambda/utils/logger'
+
+describe Datadog::Metrics::Client do
```

Comment:
We should write a test that asserts that we initialized `Datadog::Statsd.new(URL, PORT, single_thread: true)` with the right parameters.

Reply:
Because `#new` is private, this can be done as an `.instance` test.

Reply:
Yup, sounds about right, thank you for catching that!